graph/ui-tooltips/src/lib/property-info-tooltip.tsx

```diff
@@ -81,6 +83,20 @@ const PROPERTY_INFO_TOOLTIP_TYPE_OPTIONS: Record<
       "The nx-release-publish target is used to publish your project with nxrelease. Don't invoke this directly - use nx release publish instead.",
     docsUrl: 'https://nx.dev/nx-api/nx/documents/release',
   },
+  'atomizer-cloud': {
+    heading: 'Atomizer',
+    description:
+      'This target has been automatically split into multiple targets by file.',
+    docsUrl:
+      'https://nx.dev/ci/features/split-e2e-tasks#automatically-split-e2e-tasks-by-file-atomizer',
+  },
+  'atomizer-no-cloud': {
+    heading: 'Atomizer',
+    description:
+      'This target has been automatically split into multiple targets by file. We recommend running this only with distribution enabled via Nx Cloud to avoid potential performance overheads.',
```

Comment:
The text here appear title-cased, I guess there is styling applied to the tooltip that needs to be fixed.

Reply:
I would not mention the negatives (e.g. avoiding performance overheads). Focus on the value so users can try it a decide for themselves whether it makes sense for them or not. Something like:

```
Nx automatically split this potentially slow target into smaller targets by file that are faster to run. We recommend enabling Nx Cloud to benefit from better distribution, and more granular caching and flaky task re-runs.
```

Maybe @isaacplmann can fix the wording. I think we should mention that atomized targets make the other three features better (distribution, caching and re-runs).